install/kubernetes/cilium/values.yaml

```diff
@@ -2099,6 +2099,11 @@ clustermesh:
     #   tls:
     #     cert: ""
     #     key: ""
+    # # -- path to the cluster client certificate, private key and certificate authority.
```

Comment:
Could you expand on this comment a bit, e.g. saying that it is required that the user already mounted those files inside the Cilium pod (since it's not clear without looking at the commit description)?

It's also not clear to me how those files get mounted into the pod (unless you use `/var/run/cilium`). Would it make sense to also add hostPath volume mounts?